sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -381,6 +380,7 @@ public abstract partial class ClientOptions
         protected ClientOptions(Azure.Core.DiagnosticsOptions? diagnostics) { }
         public static Azure.Core.ClientOptions Default { get { throw null; } }
         public Azure.Core.DiagnosticsOptions Diagnostics { get { throw null; } }
+        public Azure.Core.Serialization.RawContentOptions RawContent { get { throw null; } }
```

Comment:
What if some properties in the schema as pascalcased and some camelcased? If we stored the schema in the client, it could handle such cases and there would not be a need for the user to set any options.

Reply:
> What if some properties in the schema as pascalcased and some camelcased? 

If that's the case, I would recommend the user not pass a configuration option to change the casing but instead use the default "strict-casing" setting to handle this precisely.  Or, if there was a reason to set PropertyNamingConvention.CamelCase, they could use the indexer syntax to override any magic we're doing.

>  If we stored the schema in the client, it could handle such cases and there would not be a need for the user to set any options.

It's true.  I think taking this approach sets us up to add that feature in the future, if we decide it was worth the investment.  I'm not sure we'd want to push out the release schedule to work on this feature before releasing the type, though.  Let me know if you feel strongly otherwise.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-net/pull/36887